components/ManageDao/TransferTokensButton.tsx

```diff
@@ -10,41 +10,40 @@ import {
 import { useWallet } from '@solana/wallet-adapter-react';
 import { Transaction } from '@solana/web3.js';
 import { useAutocrat } from '../../contexts/AutocratContext';
-import { useTokens } from '../../hooks/useTokens';
 import { useTransactionSender } from '../../hooks/useTransactionSender';
 
 export default function TransferTokensButton() {
   const wallet = useWallet();
-  const { tokens } = useTokens();
-  const { autocratProgram, daoTreasury } = useAutocrat();
+  const { autocratProgram, daoTreasury, daoTokens } = useAutocrat();
+  if (!daoTokens) return <Loader />;
+  const tokens = daoTokens;
   const sender = useTransactionSender();
 
   const handleTransfer = useCallback(async () => {
-    if (!tokens.meta?.publicKey || !wallet?.publicKey || !daoTreasury) {
+    if (!daoTokens || !wallet?.publicKey || !daoTreasury) {
       return;
     }
     const txs = [
       new Transaction()
         .add(
           createAssociatedTokenAccountIdempotentInstruction(
             wallet.publicKey,
-            getAssociatedTokenAddressSync(tokens.meta.publicKey, daoTreasury, true),
+            getAssociatedTokenAddressSync(tokens.token.publicKey, daoTreasury, true),
```

Comment:
These need to be cleaned up as I was working with a partial standard, I've now done a full one :)